deploy/crds/saas.3scale.net_v1alpha1_zync_cr.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: saas.3scale.net/v1alpha1
+kind: Zync
+metadata:
+  name: example
+spec:
+  secret:
+    vaultPath: secret/data/openshift/cluster-example/3scale/zync
+  zync:
+    replicas: 1
+    resources:
+      limits:
+        cpu: "1"
+    env:
+      dbWaitSleepSeconds: 1
+      railsEnv: staging
+  que:
+    replicas: 1
+    env:
+      railsEnv: staging
```

Comment:
Can we update the example CR railsEnv to `example` for example? (it is used on OLM doc)

Reply:
Removed in https://github.com/3scale/saas-operator/pull/5/commits/eb42a737fe50755314c40253ec3517043f04c9cd